internal/resources/slo/data_source_slo_test.go

```diff
@@ -0,0 +1,23 @@
+package slo_test
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/grafana/terraform-provider-grafana/internal/testutils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceSlo(t *testing.T) {
+	testutils.CheckCloudInstanceTestsEnabled(t)
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: testutils.ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config:      testutils.TestAccExample(t, "data-sources/grafana_slo/data-source.tf"),
+				ExpectError: regexp.MustCompile(`No SLOs Exist`),
```

Comment:
hmm, I feel like this datasource would be a bit hard to use. Normally, datasources that query a list of objects will return an empty list if none are found. Then users can do whatever they want with that value

Reply:
Modified this - the datasource just returns an empty list if none found! Thanks.